unobot.py

```diff
@@ -462,14 +463,14 @@ def get_card(self):
     @staticmethod
     def create_deck():
         new_deck = []
-        for card in COLORED_CARD_NUMS:
+        for card in (COLORED_CARD_NUMS + COLORED_CARD_NUMS[1:]):
```

Comment:
With this in place, it would be good to prove that the deck is now correct according to the rules. Should be proven by debug output as well as code analysis.